src/main/drivers/rx/rx_sx1280.c

```diff
@@ -61,9 +66,67 @@ static bool sx1280PollBusy(void)
     return true;
 }
 
-bool sx1280Init(IO_t resetPin, IO_t busyPin)
+#ifndef POLL_RX_BUSY
+typedef struct busyIntContext_s {
+    extiCallbackRec_t exti;
+    // Function to call when busy goes low
+    volatile void (*waitingFn)(void);
+} busyIntContext_t;
+
+static busyIntContext_t busyIntContext;
+
+static void sx1280BusyExtiHandler(extiCallbackRec_t *cb)
 {
+    busyIntContext_t *busyContext = container_of(cb, busyIntContext_t, exti);
+
+    // The waitingFn should call sx1280ClearBusyFn() so the ISR is not called unnecessarily
+    if (IORead(busy)) {
+        pinioSet(0, 1);
+    } else {
+        pinioSet(0, 0);
+    }
+
+    if (busyContext->waitingFn) {
```

Comment:
I had planned to only enable the interrupt when needed to avoid wasting time handling superfluous ISRs, however I'm now coming to the opinion that I will care about every transition, but clearing `busyContext->waitingFn` is a cheap way of ensuring that I don't call a handler when I shouldn't.